examples/atari_deepRL_saliency.ipynb

```diff
@@ -457,21 +457,6 @@
     "\n",
     "print_lock = threading.Lock() # Lock to control thread printing\n",
     "\n",
-    "\n",
-    "def postprocess_sal(sal, ff):\n",
-    "    \"\"\"\n",
-    "    Helper function for post-processing saliency maps, which includes resizing\n",
-    "    the saliency maps to (160,160) and multiplying them by a \"fudge factor\" ff\n",
-    "    as in the original implementation.\n",
-    "    \"\"\"\n",
-    "    pmax = sal.max()\n",
-    "    sal = cv2.resize(src=sal, dsize=(160,160)).astype(np.float32)\n",
-    "    sal -= sal.min()\n",
-    "    sal = ff * pmax * sal / sal.max()\n",
-    "    \n",
-    "    return sal\n",
-    "\n",
-    "\n",
     "class PolicyScoring(GenerateClassifierConfidenceSaliency):\n",
     "    \"\"\"\n",
     "    This saliency implementation transforms black-box action and value scores\n",
```

Comment:
We need a new implementation here since the saliency algorithm (proposed by Greydanus et al., '18) takes a squared difference between reference and perturbed scores, which is something that doesn't currently exist in our other classifier implementations.

Reply:
To quote the authors:

> We are interested in answering the question, “How much
does removing information from the region around location
(i, j) change the policy?"

They wanted to visualize the overall effect without discerning between positive and negative. They chose squared different for this.